.drone.star

```diff
@@ -1933,6 +1948,9 @@ def runWebuiAcceptanceTests(suite, alternateSuiteName, filterTags, extraEnvironm
             "from_secret": "sauce_access_key",
         }
 
+    if (earlyFail):
+        environment["EARLY_FAIL"] = "true"
```

Comment:
why do we need to set the env. variable?

Reply:
Fixed now